### PR TITLE
password label translation from languages

### PR DESCRIPTION
**Pull request by Valpuia**
This PR changes the `Password` label in lockscreen from language file

### CODE REVIEW

File: resources/lang/en/default.php [hunk 1]
```diff
@@ -3,6 +3,7 @@
 return [
     'heading' => 'Lock Screen',
     'user_menu_title' => 'Lock Screen',
+    'password' => 'Password',
```
Comment (MarJose123) [inline resources/lang/en/default.php:6]: Can you change the Key name to "password_field" to able to identify that it is use for the the field.

**Review by MarJose123 — CHANGES_REQUESTED** [review]
Can you change the Key name to "password_field" to able to identify that it is use for the the field.

File: resources/lang/en/default.php (hunk 1 above)
Comment (Valpuia) [reply resources/lang/en/default.php]: Done.. now `password` is under `fields`

**Review by MarJose123 — APPROVED** [review]

**Comment by MarJose123** [comment]
Thanks! 🥂

**Pull request merged**